var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -207,6 +207,10 @@ def setup_dependent_package(self, module, dep_spec):
         self.spec.mpicxx = join_path(self.prefix.bin, 'mpicxx')
         self.spec.mpifc  = join_path(self.prefix.bin, 'mpif90')
         self.spec.mpif77 = join_path(self.prefix.bin, 'mpif77')
+        if '+slurm' in self.spec:
+            self.spec.run = join_path(self.prefix.bin, 'srun')
```

Comment:
@alalazo that would also do. I just though storing a path might be more flexible in case some packages require the path to `mpirun` or alike during configure stage.